common/src/main/java/com/wynntils/screens/WynntilsMenuScreenBase.java

```diff
@@ -91,4 +91,12 @@ protected void renderDescription(PoseStack poseStack, String description) {
                         HorizontalAlignment.Left,
                         FontRenderer.TextShadow.NONE);
     }
+
+    protected float getTranslationY() {
+        return (this.height - Texture.QUEST_BOOK_BACKGROUND.height()) / 2f;
+    }
```

Comment:
`Translation` being used clashes with the already present usage of translating text into languages. Perhaps `translated` might work?

Reply:
Perhaps, although I think it is pretty clear what this does.